SearchHistory/tests.py

```diff
@@ -2,14 +2,14 @@
 from datetime import datetime
 import pytest
 
-PROFESSIONAL_ID = 101
-CLIENT_ID = 601
+PROFESSIONAL_ID = "professional1"
+CLIENT_ID = "client1"
```

Comment:
I don't see that you use PROFESSIONAL_ID and CLIENT_ID...

Reply:
> I don't see that you use PROFESSIONAL_ID and CLIENT_ID...

Yes, you can delete them Ofir

Reply:
Done